resources/js/components/Listing/Filters/SelectedFiltersValues.vue

```diff
@@ -0,0 +1,56 @@
+<script>
+import { swatches } from '../../../stores/useSwatches'
+
+export default {
+    props: {
+        selectedValues: {
+            type: Object,
+            default: () => {},
+        },
+    },
+
+    render() {
+        return this.$scopedSlots.default(this)
+    },
+
+    computed: {
+        activeFilters() {
+            return Object.keys(this.selectedValues)
+                .filter((filterKey) => {
+                    let { showFilter, value } = this.selectedValues[filterKey]
+                    return showFilter && (Array.isArray(value) ? value.length : !!value)
+                })
+                .reduce((result, filterKey) => {
+                    let { value } = this.selectedValues[filterKey]
+
+                    if (filterKey === 'category') {
+                        return result.concat({ code: filterKey, value: config.subcategories[value] })
+                    }
+
+                    if (filterKey === 'price') {
```

Comment:
Is this only possible with a price filterkey or is this something you can choose yourself, wether you want it to be a min/max filter?